pkgs/development/tools/vcstool/default.nix

```diff
@@ -1,21 +1,28 @@
-{ stdenv, fetchurl, python35Packages }:
+{ stdenv, python3Packages
+, git, bazaar, subversion }:
 
-  python35Packages.buildPythonApplication rec {
-    name = "vcstool";
-    version = "0.1.31";
-    #no tests
-    doCheck = false;
+with python3Packages;
 
-    propagatedBuildInputs = with python35Packages; [ pyyaml ];
-    src = fetchurl {
-        url = "mirror://pypi/v/vcstool/${name}-${version}.tar.gz";
-        sha256 = "0n2zkvy2km9ky9lljf1mq5nqyqi5qqzfy2a6sgkjg2grvsk7abxc";
-    };
+buildPythonApplication rec {
+  name = "${pname}-${version}";
+  pname = "vcstool";
+  version = "0.1.31";
 
-    meta = with stdenv.lib; {
-        description = "Provides a command line tool to invoke vcs commands on multiple repositories";
-        homepage = "https://github.com/dirk-thomas/vcstool";
-        license = licenses.asl20;
-        maintainer = with maintainers; [ sivteck ];
-    };
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0n2zkvy2km9ky9lljf1mq5nqyqi5qqzfy2a6sgkjg2grvsk7abxc";
+  };
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  makeWrapperArgs = ["--prefix" "PATH" ":" "${stdenv.lib.makeBinPath [ git bazaar subversion ]}"];
```

Comment:
I prefer this over patching, since we have wrappers anyway and it is less like to break in future.